content/scalers/azure-storage-blob.md

```diff
@@ -0,0 +1,73 @@
++++
+fragment = "content"
+weight = 100
+title = "Azure Storage Blob"
+background = "light"
++++
+
+Scale applications based on Azure Storage Blobs.
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `azure-blob` trigger for Azure Storage Blob.
+
+```yaml
+triggers:
+  - type: azure-blob
+    metadata:
+      containerName: functions-blob # Required: Container name
+      blobCount: '5' # Optional. Blob count target for HPA. Default: 5 blobs 
```

Comment:
What does this mean? Amount of blobs to scale on or?

Maybe this is more clear.
```suggestion
      blobCount: '5' # Optional. Amount of blobs to scale out on. Default: 5 blobs 
```